azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -1464,6 +1458,15 @@ public FlowRunnerEventListener() {
       metaData.put("startTime", String.valueOf(flow.getStartTime()));
       metaData.put("submitTime", String.valueOf(flow.getSubmitTime()));
 
+      // project upload info
+      final ProjectFileHandler handler = flowRunner.projectFileHandler;
```

Comment:
Why we're not using a getter to get the projectFileHandler?

Reply:
As it is accessible directly. There is similar usecase in line 1479 for sharedProps object.